src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeEmailSender.kt

```diff
@@ -11,49 +13,113 @@ class OneTimeCodeEmailSender(
 ) {
 
   private companion object {
-    val TEXT_BODY_TEMPLATE: String = """
+    fun textBodyTemplate(oneTimeCode: String, codeExpiry: String): String = """
       Your code for Send legal mail to prisons
       
       You need a code to start using Send legal mail to prisons.
 
       Your confirmation code is:
-      %s
+      $oneTimeCode
 
-      You have %s to use the code. You can only use it once.
+      You have $codeExpiry to use the code. You can only use it once.
 
       If you didn't request a code you can ignore this email.
       
       From Send legal mail to prisons team
       Ministry of Justice
     """.trimIndent()
 
-    val HTML_BODY_TEMPLATE: String = """
-      <html>
-      <head></head>
-      <body>
-        <p>
-          <strong>Your code for Send legal mail to prisons</strong>
-        </p>
-        <p>
-          You need a code to start using Send legal mail to prisons.
-        </p>
-        <p>
-          Your confirmation code is:<br/>
-          <strong>%s</strong>
-        </p>
-
-        <p>
-          <strong>You have %s to use the code. You can only use it once.</strong>
-        </p>
-
-        <p>If you didn't request a code you can ignore this email.</p>
-
-        <p>
-          From Send legal mail to prisons team<br/>
-          Ministry of Justice
-        </p>
-      </body>
-      </html>
+    fun htmlBodyTemplate(oneTimeCode: String, codeExpiry: String): String = """
+<html><head>
+    <meta http-equiv="Content-Type" content="text/html charset=UTF-8">
+    <title>Page Title</title>
+    <style media="screen">
+        body,html{
+            margin:0;
+            padding:0;
+        }
+    </style>
+</head>
+<body>
+<table style="width:100%;background-color:#dee0e2;margin:0" cellspacing="0" cellpadding="0" border="0">
+    <tbody>
+    <tr>
+        <td style="font-family:Helvetica,Arial,sans-serif;" align="center">
+            <table style="background-color:#dee0e2;margin:30px auto;max-width:600px;overflow:hidden;" cellspacing="0" cellpadding="0">
+                <tbody>
+                <tr>
+                    <td style="font-family:Helvetica,Arial,sans-serif">
+                        <table style="background:#000000" width="100%" cellspacing="0" cellpadding="0">
+                            <tbody>
+                            <tr>
+                                <td style="font-family:Helvetica,Arial,sans-serif;padding:4px 0;border-bottom:solid 10px #005ea5" width="30">
+
+                                    <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" style="font-size:24px;font-weight:bold;text-decoration:none;color:#ffffff" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.gov.uk/&amp;source=gmail&amp;ust=1648555909470000&amp;usg=AOvVaw2uN4hTGDbOHXGNCBJLkYyo">
+                                        <img src="cid:crest.png" alt="" style="margin:6px 6px 0 20px" class="CToWUd" width="30">
```

Comment:
'cid:' prefix - ah, yes, now I remember; thats how you reference a mime part. Nice 👍